test/auto_contiguous_test.cpp

```diff
@@ -174,8 +176,11 @@ TEST_CASE(standard_reshape)
         auto data = m2.add_parameter("2x2", {migraphx::shape::float_type, {2, 3, 4, 5}});
         auto add  = m2.add_instruction(migraphx::make_op("add"), data, data);
         auto ca   = m2.add_instruction(migraphx::make_op("contiguous"), add);
-        auto r    = m2.add_instruction(migraphx::make_op("reshape", {{"dims", {2, 1, 12, 5}}}), ca);
-        m2.add_return({r});
+        // extra contiguous coming from reshape logic which has "requires_std_shape" attribute
+        auto cb = m2.add_instruction(migraphx::make_op("contiguous"), ca);
```

Comment:
We shouldnt have two contiguous. Maybe we should check that the outputs are all `contiguous` operators before inserting it in.